src/gmt_io.c

```diff
@@ -4646,8 +4646,8 @@ GMT_LOCAL int gmtio_get_precision_width (struct GMT_CTRL *GMT, double x) {
 GMT_LOCAL void gmtio_check_abstime_format (struct GMT_CTRL *GMT, struct GMT_DATASET *D) {
 	bool abstime_found = false;
 	unsigned int col, row;
-	int w_max = 0, this_w;
-	double sub;
+    int w_max = 0, this_w;
+    double sub, sub_max = 0;
```

Comment:
Just a reminder: you're using spaces while @PaulWessel uses tabs for indentations.

Reply:
Ah yes, that problem.... Yes, lets stick to tabs and developers can configure their editor to give whatever width they like in the viewer

Reply:
We all use <tabs> only for indentation

Reply:
Thanks for the reminder @seisman